tool-box/gtm-jssdk-template/index.js

```diff
@@ -0,0 +1,26 @@
+var Treasure = require("td-js-sdk");
+
+var td = null;
+
+module.exports = {
+  trackPageview: function (table) {
+    td.trackPageview(table);
+  },
+  setSignedMode: function () {
+    td.setSignedMode();
+  },
+  setAnonymousMode: function () {
+    td.setAnonymousMode();
+  },
+  setGlobalId: function () {
+    td.set("$global", "td_global_id", "td_global_id");
+  },
+  fetchGlobalID: function () {
+    td.fetchGlobalID();
+  },
+  init(options) {
+    console.log("init:", options);
```

Comment:
Not sure console.log is stripped in webpack. If yes, or it is intended to have console.log here then nevermind